jet/hazelcast-3-connector/README.md

```diff
@@ -0,0 +1,138 @@
+= Hazelcast 3 Connector Sample
+
+This example shows how to use the Hazelcast 3 connector to transfer data
+between Hazelcast version 3.x and Hazelcast 5.x+ and back, or use 
+Hazelcast version 3.x for enrichment.
+
+The purpose of the Hazelcast 3 connector is to provide a way to evaluate
+Hazelcast 5 features on your existing data present in Hazelcast 3 
+cluster.
+
+In this demo we will do the following:
+- Start Hazelcast 3.x using the `Hz3MemberWithData` class - this member 
+represents an existing cluster with your data. The class loads tickers 
+(stock symbols) on startup.
+
+- Run a job `CopyMapFromHz3Example` which copies the data from a map in 
+Hz 3 cluster into a map in 5.x cluster
+
+- Start an empty Hazelcast 3.x cluster using `Hz3MemberNoData` class.
+
+- Run a job `WriteBackMapToHz3Example` which copies data from a map in 
+Hazelcast 5 cluster back to Hazelcast 3 cluster
+
+- Run a job `EnrichUsingMapHz3Example` which uses a map in Hazelcast 3 
+cluster to enrich data in a stream job running in Hazelcast 5 cluster.
+
+== Start Hazelcast 3
+
+Start Hazelcast 3 member instance by running the main method of 
+`com.hazelcast.samples.jet.hz3member.Hz3MemberWithData` class either
+from your IDE or from the command line - in the hazelcast-3-member 
+module run the following:
```

Comment:
I would propose "in the hazelcast-3-member  module" change to "in ""/hazelcast-code-samples/jet/hazelcast-3-connector/hazelcast-3-member" directory"